rest-api/Dockerfile

```diff
@@ -11,14 +11,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# To build, first build menas and be in enceladus/menas folder. Then run:
-# $> docker build -t menas .
+# To build, first build rest-api and be in enceladus/rest-api folder. Then run:
+# $> docker build -t rest-api .
 # Run image using:
 # $> docker run \
 #       -e MENAS_MONGO_CONNECTION_STRING=mongodb://host.docker.internal:27017 \
-#       -e MENAS_MONGO_CONNECTION_DATABASE=menas \
+#       -e MENAS_MONGO_CONNECTION_DATABASE=rest_api \
```

Comment:
Are you sure about this? Since the `application.properties` has not changed, I assume that the database name is kept (`menas`), so the db name should stay here.

Or do we plan to rename the database, too? (I probably wouldn't unless it was necessary, because then there might need to be a migration script etc.)

Reply:
I forgot this one. I will revert this change